config/initializers/datadog.rb

```diff
@@ -30,5 +30,36 @@
     # Enable ASM
     c.appsec.enabled = true
     c.appsec.instrument :rails
+
+    # Add dogstatsd_mapper_profiles field
+    c.dogstatsd_mapper_profiles = [
+      {
+        name: 'sidekiq',
+        prefix: 'sidekiq.',
+        mappings: [
+          {
+            match: 'sidekiq.sidekiq.*',
+            match_type: 'regex',
+            name: 'sidekiq.$1'
+          },
+          {
+            match: 'sidekiq.jobs.*perform',
+            match_type: 'regex',
+            name: 'sidekiq.jobs.perform',
+            tags: {
+              worker: '$1'
+            }
+          },
+          {
+            match: 'sidekiq.jobs.*',
```

Comment:
Why did we drop count/success/failure here?

Reply:
Because parentheses are not allowed, per to the wildcard regex pattern hardcoded in the agent

Reply:
Bummer.